animal-sniffer-maven-plugin/src/it/jdk14-with-sig14-from-cli/pom.xml

```diff
@@ -92,13 +92,6 @@
             <goals>
               <goal>check</goal>
             </goals>
-            <configuration>
```

Comment:
If you leave this in, Maven reports an error about the `signature` element not accepting children. I do not really understand what the issue here is, but apparently “flattened” CLI arguments cannot override structured POM arguments.